doc/ref_cert/lfn/chapters/chapter02.md

```diff
@@ -11,41 +11,176 @@
+  * Test to ensure custom flavours against submitted request for respective tenants
+
+**Exit criteria**: NFVI certification testing should completed with following exit criteria:
+* All mandatory test cases should pass.
+* Test results collated, centralized, and normalized, with a final report generated showing status of the test scenario/case (e.g. Pass, Fail, Skip, Measurement Success/Fail, etc), along with traceability to a functional, or non-functional, requirement.
 
 <a name="2.7"></a>
 ## 2.7 Frameworks
-
-List out the various frameworks requirements needed for certifications
+The NFVI certification framework deals with the process of testing NFVI in below three areas:
+* Compliance: The NFVI needs to comply to CNTT RA1/RA2.
+* Validation: Validation deals with the ability of NFVI to respond to Cloud APIs and interfaces.
+* Performance: Performance deals with running tests on NFVI depending on the NFVI profile and collecting KPIs.
+
+The NFVI KPIs are compared with Golden KPIs, which serve as a reference for NFVI certification. If NFVI KPIs meet Golden KPIs, NFVI is certified and granted a certification badge. If NFVI KPIs do not meet Golden KPIs, no certification badge is provided.
+
+<a name="2.7.1"></a>
+### 2.7.1 Testing
+Testing for NFVI certification falls under three broad categories - Compliance, Validation and Performance. Target NFVI for certification needs to pass all these tests in order to obtain the certification badge.
+
+<a name="2.7.1.1"></a>
+#### 2.7.1.1 Test Categories
+The following five test categories have been identified as minimal testing required to verify NFVI interoperability to satisfy the needs of VNF developer teams.
+ * Baremetal validation: To validate control and compute nodes hardware
+ * VNF Interoperability: After VNFs are on-boarded, openstack resources like Tenant, Network (L2/L3), CPU Pining, security policies, Affinity anti-affinity roles and flavors etc. would be validated.
+ * Compute components: Validate VMs status and connectivity result after performing each of listed steps. Best candidate for this testing would be identify compute node that holds VMs which has L2 and L3 connectivity.
+ * Control plane components: Validations for RabbitMQ, Ceph, MariaDB etc. and OpenStack components like Nova/Glance/Heat etc. APIs.
+ * Security: Validation for use RBAC roles and user group policies. See [Chapter 7](./chapter07.md) for complete list.
+
+<a name="2.7.1.2"></a>
+#### 2.7.1.2 Test Results
+Test results reporting will be communicated as a boolean (pass/fail). The pass/fail determination for performance-based test cases will be made by comparing NFVI KPIs (obtained after testing) with the Golden KPIs. Some of the examples of performance KPIs include, but not limited to: TCP bandwidth, UDP throughput, Memory latency, Jitter, IOPS etc. See [Chapter 4 of RM](../../../ref_model/chapters/chapter04.md) for a complete list of metrics and requirements.
+
+
+<a name="2.7.2"></a>
+### 2.7.2 Badging
```

Comment:
@mf4716 Can you provide the badge image which I can use in this section.

Reply:
Adding two - NFVI-badge.jpg, and VNF-badge.jpg.  Please upload both to the figures directory so that the dynamic links in the document will be able to find these files. Please be sure to update the links accordingly to the image name.  **Directory:**  CNTT/doc/ref_cert/lfn/figures/ 


![VNF-badge](https://user-images.githubusercontent.com/5643130/71276333-4a3f3680-231a-11ea-81d0-cfa454c46422.jpg)
![NFVI- badge](https://user-images.githubusercontent.com/5643130/71276336-4a3f3680-231a-11ea-9df1-aee5a73d6ab5.jpg)

Reply:
Uploaded both as NFVI-badge.jpg and VNF-badge.jpg in ```../figures/```